### ISSUE TITLE
[Adaptive Cards - ExpenseReport]: Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7295]
https://microsoft.visualstudio.com/OS/_workitems/edit/36148305

**Comment by Raisul123** [comment]
As per new guideline we are enhancing the severity. so we are closing this issue and logged new bug: [[Adaptive Cards - ExpenseReport]: Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'. · Issue #8476 · microsoft/AdaptiveCards (github.com)](https://github.com/microsoft/AdaptiveCards/issues/8476)
